airbyte-integrations/connectors/source-slack/source_slack/schemas/channels.json

```diff
@@ -105,6 +114,12 @@
     "num_members": {
       "type": "integer"
     },
+    "parent_conversation": {
+      "type": ["null", "string"]
+    },
+    "pending_connected_team_ids": {
+      "type": "array"
```

Comment:
can you please describe the array items structure?

Reply:
specified type of items in array fields 👍